types/react-meteor-data/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for meteor/react-meteor-data 0.2
+// Project: https://github.com/meteor/react-packages/tree/devel/packages/react-meteor-data
+// Definitions by: Igor Golovin <https://github.com/Deadly0>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
+// tslint:disable-next-line
```

Comment:
Why do you need to disable tslint? We do not want that in new definitions. Please remove this and fix the error instead.

Reply:
If this is declaring a module `meteor/react-meteor-data`, it probably go in a new file `types/meteor/react-meteor-data.d.ts` instead of in a new package?

Reply:
@andy-ms makes sense. For some reason in the install steps for this package, I mistook them for `npm install` instead of `meteor add react-meteor-data`

Reply:
Shoud I move it under `meteor` folder?

Reply:
Yes.

Reply:
@sheetalkamat done.